src/main/java/org/testng/TestNG.java

```diff
@@ -219,6 +217,22 @@ public void setOutputDirectory(final String outputdir) {
     }
   }
 
+  /**
+   * Sets the possibility to generate TestResult attributes
+   * @param generateTestResultAttributes true / false.
+   */
+  public void setGenerateTestResultAttributes(final Boolean generateTestResultAttributes) {
```

Comment:
I think we don't need extra methods here.

Reply:
It uses in *configure* method. I did it like other same methods

Reply:
Sure but the `TestNG` class is already too fat, and I'd prefer to not add more things if possible.